client/src/components/CharacterCard.js

```diff
@@ -83,17 +62,17 @@ const CharacterCard = ({ characterId }) => {
             </ActorPrimaryInfo>
 
             {films.length > 0 && (
-              <Card.Content className='center-in-mobile' textAlign='left'>
-                <h3>Movies</h3>
+              <ActorMovies textAlign='left'>
+                <h2>Movies</h2>
                 {films.map((film) => (
                   <p key={film.id}>
                     <Link to={`/film/${film.id}`}>{film.title}</Link>
                     <br />
```

Comment:
Using a `<br>` to create a margin is a bit crude. A styled-component with some `margin` instead of a `p` would be better.